client/lib/ScriptInstance.ts

```diff
@@ -35,6 +49,29 @@ export default class ScriptInstance {
     this.sessionNameCountByName[name] += 1;
     return name;
   }
+
+  public getScriptCallsite(ignoreMode = false): string {
```

Comment:
This method name uses "Callsite" with lowercase "s" while the commons/lib/utils uses a capital "S" for callSite. We should pick one or the other.